index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that a <a>subject</a> controls, or a <a>holder</a>
+has been designated to use, one or more particular confirmation methods. This
+binds a specific <a>claim</a> about a <a>subject</a> to one or more
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+around how a <a>verifier</a> can validate that the <a>holder</a> is bound to
+the <a>claims</a> presented in a <a>verifiable credential</a>.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a
+different mechanism to validate whether the <a>holder</a> is bound to the
+presented <a>claims</a>, if required. Such decision can prevent impact on the
+<a>verifier</a>'s liability or lack thereof if not specified by other means
+such as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <div class="note">
+          <p>
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
```

Comment:
An informative statement around compatibility with RFC7800 and the "confirmation claim" could potentially be made here if desirable.

```suggestion
For example, an <a>issuer</a> can include a confidence method based on public
key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
demonstrate they are able to generate and include a <a>proof</a> with a
cryptographic signature in the <a>verifiable presentation</a> that will verify
against the verification key expressed in the confidence method in the
embedded <a>verifiable credential</a>.
```